modules/microshift-storage-classes.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-classes_{context}"]
+== Storage classes
+
+Storage classes provide the workload layer interface for selecting a device class. The following storage class parameters are supported in {product-title}:
+
+* `csi.storage.k8s.io/fstype`, which selects the filesystem types. `xfs` or `ext4` filesystem type sare supported
+* `topolvm.io/device-class` is the name of the device class. If a device class is not provided, the default device class is assumed.
+
+Multiple storage classes can refer to the same device class. You can provide varying sets of parameters for the same backing device class, such as `xfs` and `ext4` variants.
+
+.{product-title} defines the following as the default storage class:
+
+[source, yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true" <1>
+  name: topolvm-provisioner
+parameters:
+  "csi.storage.k8s.io/fstype": "xfs" <2>
+provisioner: topolvm.io <3>
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer <4>
+allowVolumeExpansion: false <5>
+----
+[.small]
+<1> This storage class is the cluster's default. If a PVC does not specify a storage class, this class is assumed. Delete “true” if you are not using the {product-title} default storage class. You must set this if you are defining a storage class.
```

Comment:
I think we want them to delete the annotation entirely, not just the `"true"` part.

```suggestion
<1> This storage class is the cluster's default. If a PVC does not specify a storage class, this class is assumed. Delete the `storage class.kubernetes.io/is-default-class` annotation if you are not using the {product-title} default storage class. You must set this if you are defining a storage class.
```

Reply:
Used Jon's suggestion that follows.